.ci/watchdog/SlackCommunicator.py

```diff
@@ -27,15 +27,15 @@
 import slack
 import os
 
-_CI_ALERTS_CHANNEL = 'ngraph-onnx-ci-alerts'
```

Comment:
For now we want messages to reach both Slack and Teams. Let's not change the Slack channel.